### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by rafixer**

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+2.0.0-rc.1 / 2019-02-21
```
Comment (rafixer) [inline History.md:1]: - convert to CHANGELOG.md
- describe 2.0.0 version changes

File: Readme.md [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
 
-  A node.js library to run cpu-intensive tasks in a separate processes and to not block the event loop.
+  A node.js library to run cpu-intensive tasks in a separate processes and to not to block the event loop.
 
 
 ## Installation
 
 ```bash
-$ npm install worker-nodes
+$ npm install worker-nodes@next
```
Comment (rafixer) [inline Readme.md:12]: remove next

File: Readme.md [hunk 3]
```diff
@@ -205,18 +202,18 @@ Example results:
...
-worker-nodes@1.2.0              1521            646            528              641              367           272              119       0
-workerpool@2.1.0               12055           7356           5726              896              212           731               74       0
-worker-farm@1.3.1              12124           6711           5501             1577              446           689               74       0
-process-pool@0.3.4             12348           6866           5474             1696              458           698               76       0
-worker-pool@3.0.2              14029           7633           5604             2285              649           769              104       0
-
-os : Darwin / 15.5.0 / x64
-cpu : Intel(R) Core(TM) i7-4578U CPU @ 3.00GHz × 4
-node : 6.9.1 / v8: 5.1.281.84
+name                 time: total [ms]  time usr [ms]  time sys [ms]  worker usr [ms]  worker sys [ms]  mem rss [MB]  worker rss [MB]  errors
```
Comment (rafixer) [inline Readme.md:205]: update results

File: docs-src/Readme.hbs [hunk 4]
```diff
@@ -10,10 +10,10 @@
 ## Installation
 
 ```bash
-$ npm install worker-nodes
+$ npm install worker-nodes@next
```
Comment (rafixer) [inline docs-src/Readme.hbs:13]: next

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "worker-nodes",
-  "version": "1.6.1",
+  "version": "2.0.0-rc.1",
```
Comment (rafixer) [inline package.json:3]: 2.0.0

**Comment by bgalek** [comment]
LGTM :)

**Pull request merged**